web/src/user_profile.ts

```diff
@@ -1250,7 +1250,7 @@ export function initialize(): void {
     });
 
     $("body").on("input", "#user-profile-streams-tab .stream-search", () => {
-        const $input = $<HTMLInputElement>("input#user-profile-streams-tab .stream-search");
+        const $input = $<HTMLInputElement>("#user-profile-streams-tab .stream-search");
```

Comment:
Should it be `input.stream-search`?

Reply:
Ah hmm, perhaps that's what the author wanted to do in 19b85eb60075e67a03eb90a1b4e861d879ff2d57. Updated.